specification/webdriver-classic.bs

```diff
@@ -0,0 +1,177 @@
+<pre class="metadata">
+Title: webdriver-classic
+Shortname: wecg-webdriver-classic
+Level: 1
+Group: wecg
+Status: w3c/CG-DRAFT
+URL: https://w3c.github.io/webextensions/specification/webdriver-classic.html
+Editor: Kiara Rose, Apple, kiara_rose@apple.com
+Abstract: This specification defines how web extensions should be loaded and unloaded in WebDriver Classic.
+Repository: w3c/webextensions
+</pre>
+
+  <section>
+    <h3> Endpoints</h3>
+
+    <table class=simple>
+     <tr>
+      <td>POST</td>
+      <td>/session/{<var>session id</var>}/webextension</td>
+      <td>Load WebExtension</td>
+     </tr>
+
+     <tr>
+      <td>DELETE</td>
+      <td>/session/{<var>session id</var>}/webextension/{<var>extension id</var>}</td>
+      <td>Unload WebExtension</td>
+     </tr>
+    </table>
+  </section>
+
+  <section>
+    <h3>Errors</h3>
+    <table class=simple>
+     <tr>
+      <th>Error Code
+      <th>HTTP Status
+      <th>JSON Error Code
+      <th>Description
+     </tr>
+
+     <tr>
+      <td>no such web extension
+      <td>404
+      <td><code>no such web extension</code>
+      <td>No web extension matching the given extension id was found
+          amongst the <var>session</var>&apos;s loaded extensions.
+     </tr>
+
+      <tr>
+       <td>unable to load extension
+       <td>500
+       <td><code>unable to load extension</code>
+       <td>A command to load an extension could not be satisfied.
+     </tr>
+
+     <tr>
+       <td>unable to unload extension
+       <td>500
+       <td><code>unable to unload extension</code>
+       <td>A command to unload an extension could not be satisfied.
+     </tr>
+    </table>
+  </section>
+
+  <section>
+    <h3>Commands</h3>
+    <section>
+      <h2>WebExtensions</h2>
+      <p>The WebExtensions API provides an interface that allows
+      extensions to modify and enhance the capability of the browser.
+      This section describes the interaction with WebExtensions.
+
+      <section>
+        <h3>Load WebExtension</h3>
+        <table class="simple jsoncommand">
+         <tr>
+          <th>HTTP Method</th>
+          <th>URI Template</th>
+         </tr>
+         <tr>
+          <td>POST</td>
+          <td>/session/{<var>session id</var>}/webextension</td>
+         </tr>
+        </table>
+
+        <p>The remote end steps, given <var>session</var>, <var>URL
```

Comment:
It would be nice to mention that this is based on the BiDi algorithm. We could maybe even say that we are mimicking the "remote end steps" from here: https://www.w3.org/TR/webdriver-bidi/#command-webExtension-install